gaphor/storage/storage.py

```diff
@@ -112,11 +111,11 @@ def save_canvas(value):
 
         The extra attribute reference can be used to force UML
         """
-        assert isinstance(value, gaphas.Item)
```

Comment:
Would it make more sense now to rename this function and the one below it to `save_diagram` and `save_diagram_item` now?

Reply:
Done